hack/bumper.sh

```diff
@@ -0,0 +1,69 @@
+#!/bin/bash -e
+
+# package bump helper
+# make sure you already have a local repo with remote upstream, and origin branches
+# no untracked files allowed on folder
+# it does reset --hard so beware, also pushes to save time
+
+# ./hack/bumper.sh CVE-2021-38561 golang.org/x/text@v0.3.7 release-0.89
+
+# to skip semver check (for example in case package is dropped from go.mod after updating)
```

Comment:
I don't get the scenario where SKIP would be true.
Why would we bump a package, and it would end up getting removed?

Reply:
Nice question
for example if there was a static `replace`, which the script removes 
https://github.com/kubevirt/cluster-network-addons-operator/pull/1968/files#diff-958372dc14cde9068a295622a0c8af96ac251e37b948589228f81ac0a7d0da6cR31

and once the script done, the new set doesn't need anymore the old set

Reply:
ah you mean there was a lonely replace without a require?

Reply:
there were both

Reply:
OK I guess. I just never encountered this scenario before.